clang/test/CodeGenSYCL/uses_aspects.cpp

```diff
@@ -28,7 +28,7 @@ template <cl::sycl::aspect Aspect>
 void func5() {}
 
 [[__sycl_detail__::__uses_aspects__(cl::sycl::aspect::cpu)]] void func6();
-// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !intel_used_aspects ![[ASPECTS1]] {
+// CHECK: define dso_local spir_func void @{{.*}}func6{{.*}} !intel_used_aspects ![[ASPECTS4:[0-9]+]] {
```

Comment:
Why did this change? `func6 `should not have 2 aspects in IR as in your change

Reply:
`func6` definition can be found down below and the function uses `Type2WithAspect`, which annotated with both `fp16` and `cpu` aspects